packages/typography/sass/font-brand.scss

```diff
@@ -1,3 +1,5 @@
+@import './font-family';
+
 @font-face {
```

Comment:
Kinda cool if `@font-face` would get outputted just _once_ when importing this file multiple times. 🤔

Reply:
Would the SASS `@use` really have any effect in our case where each _*.scss_ file is an independent compilation unit, created by the `sass-loader`? The different compilations don't have any shared state.

Reply:
I believe only `dart-sass` supports `@use` at the moment. I would heartily support a move to `dart-sass`; `node-sass`'s native code is a never-ending source of build problems for developers.

Reply:
If we don't want to use `@use` and stick to a more traditional Sass setup, we can still accomplish this goal. The idea is to separate Sass files into CSS-outputting files, and Sass-only files like `functions.scss` and `constants.scss`). Then you make sure that the Sass-only files are the only ones that can get imported in multiple places, and the CSS-outputting ones only exist as leaf nodes.

You then organise things very clearly in the library so that developers know whether you're importing something that outputs CSS or something that gives you SCSS objects.

With all that said, `@use` and friends are a much better model, and we should move to it if we can.

Reply:
Yep, I specifically wanted to place this import in separate `font-brand.scss` so that consumers can be a bit more mindful about it outputting CSS. I could add a bit about this in Readme file though.

Reply:
We'd see other benefits from `@use` as well: we sometimes import SASS files that have variables by same name; from Gutenberg base styles and in Calypso. Not only we'd avoid variable name collisions, but it would also be more clear where variables are coming from. We'll likely have more those issues once we start moving Calypso towards core components.

Reply:
Yup, namespacing is a game-changer, especially when combined with the ability to easily set options on the things you're importing (by overriding variables) with `with` (`@use './foo' with ( $bar: 'baz' )`).

For a great example of `@use` in a library, check out Google's [material-components-web](https://github.com/material-components/material-components-web).